### PR TITLE
Add riscv platform support for tflite with cmake.

### PR DESCRIPTION
**Pull request by JerryShih**
This pr try to add the riscv platform support with cmake.
It includes:
1. prebuilt binary download script for riscv clang compiler and qemu.
2. the riscv cmake toolchain file which contains the toolchain and compiling options related setting for riscv platform
3. the howto document for all build steps.

### CODE REVIEW

**Comment by JerryShih** [comment]
The qemu running result for benchmark tool:
```
/tensorflow/tensorflow/lite/tools/cmake/build$ /tensorflow/tensorflow/lite/tools/cmake/riscv/Prebuilt/qemu/linux/RISCV/bin/qemu-riscv64 \
  -cpu rv64,x-v=true,x-k=true,vlen=512,elen=64,vext_spec=v1.0 \
  -L /tensorflow/tensorflow/lite/tools/cmake/riscv/Prebuilt/toolchain/clang/linux/RISCV/sysroot \
  ./tools/benchmark/benchmark_model --help

STARTING!
usage: ./tools/benchmark/benchmark_model <flags>
Flags:
        --input_layer=                          string  optional        input layer names
        --min_nodes_per_partition=0             int32   optional        The minimal number of TFLite graph nodes of a partition that has to be reached for it to be de
        ...
```

**Comment by JerryShih** [comment]
@terryheo
Could you please review this pr? The riscv platform supporting with cmake.

**Comment by terryheo** [comment]
Thanks for your contribution!
I have few comments on this.
1. RISCV is not our primary target devices since TFLite don't have accelerated kernels
2. I wanted to remove logic to maintain toolchain from this change. As you can see from https://www.tensorflow.org/lite/guide/build_cmake_arm, CMake builds works well by providing few necessary variables. I think RISCV build should be the similar. such as,
```
RISCVCC_PREFIX=$HOME/toolchains/riscv/bin/riscv64-unknown-linux-gnu-
cmake -DCMAKE_C_COMPILER=${RISCVCC_PREFIX}gcc -DCMAKE_CXX_COMPILER=${RISCVCC_PREFIX}g++ -DTFLITE_ENABLE_XNNPACK=OFF ../tensorflow/lite/
../tensorflow/lite/
```

In summary, I'm OK to have a small section for RISCV under https://www.tensorflow.org/lite/guide/build_cmake_arm. But I'm reluctant to have a separate page for RISCV since we don't have optimized kernels.

**Comment by gbaned** [comment]
@JerryShih Can you please check @terryheo's comments and keep us posted ? Thanks!

**Comment by gbaned** [comment]
@JerryShih  Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
@JerryShih Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
